plugins/inputs/execd/execd.go

```diff
@@ -96,9 +106,6 @@ func (e *Execd) cmdReadOut(out io.Reader) {
 		}
 	}
 
```

Comment:
```suggestion
```

Reply:
Ah, thanks! I'm having a hard time understanding all the messages from the linter, I must have missed this one